app/assets/javascripts/redux/modules/models/responses.js

```diff
@@ -40,4 +69,39 @@ export const reducer = handleActions({
     loading: false,
     loaded: false,
   }),
+
+  [LISTEN_SIGNALS_PUT_REQUEST]: (state, action) => ({
+    ...state,
+    loading: true,
+    loaded: false,
+  }),
+
+  [LISTEN_SIGNALS_PUT_REQUEST_SUCCESS]: handleResponseRequest,
+
+  [LISTEN_SIGNALS_PUT_REQUEST_FAIL]: (state, action) => ({
+    ...state,
+    error: action.payload,
+    loading: false,
+    loaded: false,
+  }),
+
+  [LISTEN_SIGNALS_POST_REQUEST]: (state, action) => ({
```

Comment:
Nvm...I'm dumb. Here you actually don't need to do anything. No reducer function is needed here for this action.